### PR TITLE
Fix small mistake in description

### PR DESCRIPTION
**Pull request by peacekeeper**

### CODE REVIEW

**Review by OR13 — APPROVED** [review]

**Review by mkhraisha — APPROVED** [review]

**Review by msporny — APPROVED** [review]

**Review by OR13 — APPROVED** [review]

**Review by mkhraisha — APPROVED** [review]

**Comment by mkhraisha** [comment]
This appears to have all necessary approvals, looks ready to merge.

**Comment by OR13** [comment]
CODEOWNERS:

@peacekeeper @msporny @mavarley @OR13 @mkhraisha
 
How many days do we leave a PR with only approvals open before merging?

**Comment by peacekeeper** [comment]
> How many days do we leave a PR with only approvals open before merging?

Not sure if we have ever defined a waiting period for PRs. In this case I think it's been open long enough, and it's a ridiculously small change :) Somehow I just didn't want to be the one to merge my own PR.

**Pull request merged**
